app/src/main/java/de/mvhs/android/zeiterfassung/EditActivity.java

```diff
@@ -0,0 +1,30 @@
+package de.mvhs.android.zeiterfassung;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
+import android.support.design.widget.Snackbar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+
+public class EditActivity extends AppCompatActivity {
+
+  @Override
+  protected void onCreate( Bundle savedInstanceState ) {
+    super.onCreate( savedInstanceState );
+    setContentView( R.layout.activity_edit );
+    Toolbar toolbar = (Toolbar) findViewById( R.id.toolbar );
```

Comment:
Ja, "Empty Activity" wäre die richtige Wahl, wenn man Java-Code UND XML Layout erstellen will. In der Hausaufgabe war nur das Layout gefragt, ohne Java-Code.